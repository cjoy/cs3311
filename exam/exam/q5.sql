-- COMP3311 12s1 Exam Q5
-- The Q5 view must have attributes called (team,reds,yellows)

drop view if exists Q5;
create view Q5
as
... SQL code for view goes here ...
;
