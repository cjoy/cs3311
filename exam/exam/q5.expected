Argentina|0|3
Australia|1|6
Brazil|1|3
China|1|1
England|4|4
France|0|12
Germany|1|7
Iran|1|6
Italy|2|3
Japan|7|2
Korea|1|5
Netherlands|0|4
Spain|7|7
Sweden|1|7
Uruguay|2|8
